pyproj/__init__.py

```diff
@@ -81,8 +80,6 @@
 from pyproj.proj import Proj, pj_list, proj_version_str  # noqa: F401
 from pyproj.transformer import Transformer, itransform, transform  # noqa: F401
 
-PYPROJ_CONTEXT.set_search_paths()
```

Comment:
Does this mean that, before you do any actual operation, checking the datadir isn't yet up to date?

Or does
```
import pyproj
pyproj.datadir.get_data_dir()
```

still give the correct thing because when calling `get_data_dir` it will trigger this "search paths" if it was not done yet?

Reply:
On 2.3.1, when you call set_search_paths, it will reset the data directory for the context. On initial load, it sets the data directory to whatever is found by get_data_directory.

Reply:
Feel free to undo the resolve if you have further questions.